src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -1527,7 +1541,7 @@ subroutine read_fv3_netcdf_guess(fv3filenamegin)
            call gsi_copy_bundle(gsibundle_fv3lam_tracersmoke_nouv,GSI_ChemGuess_Bundle(it))
          endif
 
-         if(if_model_dbz) call gsi_copy_bundle(gsibundle_fv3lam_phyvar_nouv,GSI_MetGuess_Bundle(it))
+         if(if_model_dbz .or. if_model_fed) call gsi_copy_bundle(gsibundle_fv3lam_phyvar_nouv,GSI_MetGuess_Bundle(it))
```

Comment:
see previous comments about those "if ( .. or ..) " statement

Reply:
@TingLei-NOAA 
revised to: 
        if ( nphyvario3d > 0 ) then